index.js

```diff
@@ -3,7 +3,7 @@
 const fs = require('fs')
 const path = require('path')
 const steed = require('steed')
-
+const kBaseOptions = Symbol.for('base-options')
```

Comment:
Can this be namespaced? Maybe `Symbol.for('fastify-plugin-default-options')` or something?